pom.xml

```diff
@@ -28,11 +28,14 @@
 
   <groupId>com.huaweicloud</groupId>
   <artifactId>spring-cloud-huawei</artifactId>
-  <version>1.3.2</version>
+  <version>1.3.3-Hoxton-SNAPSHOT</version>
   <packaging>pom</packaging>
```

Comment:
is SNAPSHOT necessary？

Reply:
use SNAPSHOT for developing, this is a convention and can reduce problems when user's execute `mvn deploy`